server/src/main/java/org/kaazing/gateway/server/config/parse/GatewayConfigParser.java

```diff
@@ -433,37 +406,6 @@ private void validateGatewayConfig(GatewayConfigDocument configDoc, List<String>
     }
 
     /**
-     * Write out an input stream to a new file.
-     *
-     * @param in       the input stream
-     * @param fileName the file name
-     * @return whether file was successfully written
-     * @throws IOException
-     */
-    private void writeToNewFile(InputStream in, String fileName) throws IOException {
```

Comment:
the code that writes out the new config file is on line 149ish inside the translate method